libraries/mbed/targets/hal/TARGET_RENESAS/TARGET_RZ_A1H/device.h

```diff
@@ -16,6 +16,10 @@
 #ifndef MBED_DEVICE_H
 #define MBED_DEVICE_H
 
+#if defined(__arm__) && (__ARMCC_VERSION <= 5040027)
```

Comment:
Can you add a comment above these code, with an explanation why is that pragma used? So users who will come accross this, would get an explanation. I did not see anything neither in the commit message.

Looks like this is a specific bug which happens for this platform's code, so we can accept this as a workaround at the moment.